src/appendix/git.md

```diff
@@ -0,0 +1,199 @@
+# Appendix C: Version control
+
+Some early computer games had no means of saving your progress. So when you
+painstakingly dodged spear, evaded boulders, jumped crevasses and made a
+mistake, you had to start all over again. Saving your progress makes playing
+games easier. Version control is like saving your progress for projects.
+
+In this appendix we will cover git and how you can use it keep track of changes
+in your project.
+
+## Installing `git`
+
+We use the version control system [`git`][git]. In this section you can find
+details how to setup git for your system.
+
+### Linux
+
+On a Debian based Linux distribution you can install `git` with the following
+command:
+
+```sh
+$ sudo apt-get install git
+```
+
+### Mac OS X
+
+There is a binary installer for `git` that can be found at
+[http://git-scm.com/download/mac](http://git-scm.com/download/mac).
+
+### Windows
+
+I hope to provide better instructions for windows soon. If you know how consider
+[contributing][] to the book.
```

Comment:
@Charlotteis I am using the [reference style of linking](https://daringfireball.net/projects/markdown/syntax#link) here, and the shorthand variant on top of that.

The full form would be `[contributing][contributing]`. When the reference is the same as what is displayed the reference can be omitted.

You still need to point to an actual resource. I have collected several reference at the end of the document. For `contributing` I added

```text
[contributing]: https://github.com/intermezzOS/book
```